source/tasks/BuildInformation/BuildInformationV6/buildInformation.ts

```diff
@@ -0,0 +1,189 @@
+import * as tasks from "azure-pipelines-task-lib/task";
+import * as path from "path";
+import os from "os";
+import { OctoServerConnectionDetails } from "../../Utils/connection";
+import * as vsts from "azure-devops-node-api";
+import { WebApi } from "azure-devops-node-api";
+import { getInputParameters } from "./inputParameters";
+import { BuildInformationRepository, Client, ClientConfiguration, CreateOctopusBuildInformationCommand, Logger, PackageIdentity } from "@octopusdeploy/api-client";
+import { TaskWrapper } from "tasks/Utils/taskInput";
+
+export interface IOctopusBuildInformation {
```

Comment:
Are these copied from the v5 code and no longer needed? I'd expect the command structure, and the types it uses, should just come from the API client now.